apache/superset
# build: Add helm chart option to deploy sidecars with node and worker

### SUMMARY
Adds ability to deploy additional, long-lived containers along side Superset

### TESTING INSTRUCTIONS
Create a docker image for a long running script:
Dockerfile:
```
FROM python:3.8.6
COPY  entrypoint.sh ./
RUN chmod 755 ./entrypoint.sh
ENTRYPOINT /entrypoint.sh
```

entrypoint.sh:
```
while true
do
   echo "Hello"
   sleep 5
done
```

Add extraContainer definitions under SupersetWorker and SupersetNode
```
supersetNode:
  extraContainers:
    - name: dashboard-rbac-sync
      image: localhost:5001/looping-image:latest
supersetWorker:
  extraContainers:
    - name: dashboard-rbac-sync
      image: localhost:5001/looping-image:latest
```


After installing,

`k get pods`

should show 2 ready containers in the SupersetNode and SupersetWorker pods
